ClientRuntimes/NodeJS/lib/serviceClient.js

```diff
@@ -8,34 +8,48 @@ var Constants = require('./constants');
 var ProxyFilter = require('./filters/proxyfilter');
 var RedirectFilter = require('./filters/redirectfilter');
 var SigningFilter = require('./filters/signingfilter');
+var ExponentialRetryPolicyFilter = require('./filters/exponentialretrypolicyfilter');
 var requestPipeline = require('./requestPipeline');
 var utils = require('./utils');
 
 /**
-   * @class
-   * Initializes a new instance of the ServiceClient class.
-   * @constructor
-   * 
-   * @param {object} credentials - BasicAuthenticationCredentials or 
-   * TokenCredentials object used for authentication.  
-   * 
-   * @param {Array} filters
-   */
-function ServiceClient(credentials, filters) {
-  if (!filters) {
-    filters = [];
+ * @class
+ * Initializes a new instance of the ServiceClient class.
+ * @constructor
+ * @param {object} options The parameter options
+ * 
+ * @param {object} [options.credentials]    - BasicAuthenticationCredentials or 
+ * TokenCredentials object used for authentication.  
```

Comment:
my suggestion is to drop the specific derived type names. Because in near future if we support new type, or for any reason to rename, likely. This comment will become invalid without notice. Updating common is always not cheap, as we need refresh all dependents downstream.